manifests/r/RemoveBG/RemoveBG/2.2.4/RemoveBG.RemoveBG.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created with komac v2.12.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: RemoveBG.RemoveBG
+PackageVersion: 2.2.4
+PackageLocale: en-US
+Publisher: Canva Austria Gmbh
+PublisherUrl: https://www.remove.bg/
+PackageName: Remove image backgrounds 100% automatically in seconds

```

```suggestion
PackageName: remove.bg
```
This should match Control Panel